events/templates/widgets/sortOptions.html

```diff
@@ -0,0 +1,17 @@
+<!-- Toggle style sort by buttons that keeps user choice after form submission -->
+<div class="form-group form-inline btn-group btn-group-toggle" data-toggle="buttons">
+  <label class="mr-2">Sort By: </label>
+  {% for choice in form.sort %}
+  <label for="{{choice.id_for_label}}" class="btn btn-outline-primary {% if sort_by == forloop.counter %}active{% endif %}">
+    {{choice}}
+  </label>
+  {% endfor %}
+</div>
```

Comment:
remove the data-toggle attribute and change the label into a button. Then the form will submit when a button is clicked instead of clicking your selection and then clicking again on the search button.